jlcparts/jlcpcb.py

```diff
@@ -9,6 +9,9 @@
 
 class JlcPcbInterface:
     def __init__(self, key: str, secret: str) -> None:
```

Comment:
This change is completely wrong. `JlcPcbInterface` doesn't care where the key comes from. If the error should be reported anywhere, it should be the place that uses `JlcPcbInterface`. This change would only turn the code into spaghetti.

Reply:
Fair enough. The error should be higher up, I agree. Will change.